pkg/logql/lex.go

```diff
@@ -240,21 +240,27 @@ func tryScanDuration(number string, l *scanner.Scanner) (time.Duration, bool) {
 		return 0, false
 	}
 	// we've found more characters before a whitespace or the end
-	d, err := time.ParseDuration(sb.String())
+	var duration time.Duration
+	prometheusDuration, err := model.ParseDuration(sb.String())
```

Comment:
A comment here explaining why we're doing this would be helpful

Reply:
Done